src/tunacode/ui/commands/debug.py

```
@@ -0,0 +1,47 @@
+
+if TYPE_CHECKING:
+    from tunacode.ui.app import TextualReplApp
+
+
+class DebugCommand(Command):
+    """Toggle debug logging to the screen and configured file logger."""
+
+    name = "debug"
+    description = "Toggle debug logging to screen"
+
+    async def execute(self, app: TextualReplApp, args: str) -> None:
```

Ruff (0.15.0)

[warning] Unused method argument: `args`

(ARG002)